tests/e2e_tests/subcommands/root/test_root_boost_weights.py

```diff
@@ -0,0 +1,69 @@
+from bittensor.commands.root import (
+    RootSetBoostCommand,
+    RootSetWeightsCommand,
+    RootRegisterCommand,
+)
+from bittensor.commands.stake import StakeCommand
+from bittensor.commands.network import RegisterSubnetworkCommand
+from bittensor.commands.register import RegisterCommand
+from ...utils import (
+    new_wallet,
+    sudo_call_set_network_limit,
+    sudo_call_set_weight_limit,
+    sudo_call_set_min_stake,
+)
+import bittensor
+
+
+def test_root_get_set_weights(local_chain, capsys):
+    """Test case to set weights for root network"""
+
+    (wallet, exec_command) = new_wallet("//Alice", "//Bob")
```

Comment:
This line is outdated, please rebase from staging.

Reply:
will rebase it.